### DISCUSSION TITLE
Can I use one schema to validate multiple csv files?

### DISCUSSION THREAD

**Discussion by lwinfree** [frictionlessdata/frictionlessdata.io#637] [Q&A] [answered]
is there a way to override path in the yaml?
my goal is to hand someone else a description so that they can validate all their files.
(from moopekkhā on Discord)

**Comment by lwinfree** [comment]
tagging @roll to take a look :-)

**Reply by jm-moreau** [reply]
Hi there, yes I wondered if I could use frictionless to write a general description that can be used for a bunch of `.csv `files. Is there way to omit `path` in the `description.yaml` then provide it from the command line during validation?

**Comment by roll** [comment]
Yes, using the Table Schema:

```bash
$ frictionless describe --type schema table.csv > schema.yaml
$ frictionless validate --schema schema.yaml <any-similiar-table>.csv
```

**Reply by mingfang** [reply]
I wish I can do the same with the transform command
e.g. `frictionless transform --pipeline pipeline.yaml data.csv`

**Reply by roll** [reply]
Thanks @mingfang,

I created a feature request for it - https://github.com/frictionlessdata/frictionless-py/issues/895

**Comment by augusto-herrmann** [comment, marked as answer]
You can also reference the same schema in several resources of a data package,

For a working example, see [okfn/publicbodies](https://github.com/okfn/publicbodies/).

In `datapackage.json`, we reference the schema file several times:

```json
    {
      "path": "data/br.csv",
      "profile": "tabular-data-resource",
      "name": "br",
      "format": "csv",
      "mediatype": "text/csv",
      "schema": "public-body-schema.json"
    },
    {
      "path": "data/ch.csv",
      "profile": "tabular-data-resource",
      "name": "ch",
      "format": "csv",
      "mediatype": "text/csv",
      "schema": "public-body-schema.json"
    },
```

and the actual schema is on `public-body-schema.json`.
